### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by tobiscr**

### CODE REVIEW

**Review by clebs — APPROVED** [review]

**Review by ruanxin — APPROVED** [review]

**Review by cnvergence — APPROVED** [review]

**Review by clebs — APPROVED** [review]

**Review by franpog859 — APPROVED** [review]

File: pkg/db/columnhandler.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func (ch *ColumnHandler) ColumnName(field string) (string, error) {
 	return "", fmt.Errorf("entity '%s' has no field '%s': cannot resolve column name", ch.entity, field)
 }
 
-//ColumnNamesCsv returns the CSV string of the column names
+// ColumnNamesCsv returns the CSV string of the column names
```
Comment (VOID404) [inline pkg/db/columnhandler.go:157]: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
// ColumnNamesCSV returns the CSV string of the column names
```

File: pkg/db/columnhandler_test.go [hunk 2]
```diff
@@ -161,7 +161,7 @@ func splitAndTrimCsv(csv string) []string {
 	return result
 }
 
-//mock DB entity used to test the validation logic
+// mock DB entity used to test the validation logic
```
Comment (VOID404) [inline pkg/db/columnhandler_test.go:164]: https://go.dev/doc/comment
> [...] doc comments for types start with complete sentences naming the declared symbol. An explicit subject often makes the wording clearer, and it makes the text easier to search, whether on a web page or a command line

**Comment by VOID404** [comment]
As per [the style guide](https://go.dev/doc/comment), Go doc comments should use complete sentences ending with a dot

**Review by kwiatekus — APPROVED** [review]

File: pkg/db/columnhandler_test.go (hunk 2 above)
Comment (tobiscr) [reply pkg/db/columnhandler_test.go:164]: Fully agree on your points. But a fix would need to change actually all files which use comments to be consistent again. 

As we are in maintenance mode and not changing the old reconciler any more (replacement happens in coming months), we should avoid bigger code changes as the efforts will get lost soon.

Please excuse when I recommend to leave the wrong comment style and focus on clean comments in the new reconciler code bases.

File: pkg/db/columnhandler.go (hunk 1 above)
Comment (tobiscr) [reply pkg/db/columnhandler.go:157]: Agree on your points. A fix would imply to change the wrong fct-name as well to have it consistant. Please see my comments below, why we try to minimise any changes in the old reconciler (we are close to the transition phase and will drop the old reconciler in next few months).

**Review by ebensom — APPROVED** [review]

**Pull request merged**